src/main/java/uk/gov/hmcts/darts/arm/mapper/impl/AnnotationArchiveRecordMapperImpl.java

```diff
@@ -237,7 +238,8 @@ private String formatDateTime(OffsetDateTime offsetDateTime) {
     private String getHearingDate(AnnotationDocumentEntity annotationDocument) {
```

Comment:
now that these methods are returning DateTime is it worth changing the method names? e.g. getHearingDateTime()